packages/mantine/src/components/code-editor/__tests__/CodeEditor.spec.tsx

```diff
@@ -1,24 +1,26 @@
-import {useForm} from '@mantine/form';
 import {loader} from '@monaco-editor/react';
 import {render, screen, waitForElementToBeRemoved, within} from '@test-utils';
+import {useForm} from '../../../form';
 
 import {CodeEditor} from '../CodeEditor';
 import {XML} from '../languages/xml';
 
+vi.mock('monaco-editor');
+vi.mock('@monaco-editor/react');
+
 describe('CodeEditor', () => {
     beforeEach(() => {
-        jest.clearAllMocks();
+        vi.clearAllMocks();
     });
 
     it('renders the monaco editor and a copy to clipboard button', async () => {
-        jest.mock('monaco-editor');
         render(<CodeEditor label="label" description="description" />);
 
-        await waitForElementToBeRemoved(screen.queryByRole('presentation'));
+        await waitForElementToBeRemoved(screen.queryByRole('presentation'), {timeout: 5000});
```

Comment:
We still need that timeout?

Reply:
I will try it out, I just finished finnally fixing all the cypress tests